### PR TITLE
TO-DO-LIST Add&Remove

### PR DESCRIPTION
**Pull request by suzi216**
**To Do list: add & remove**

For this project I have done all these requirements:

- Created a new JavaScript files for the new functionality.

- Implemented a function for adding a new task in the array.

- Implemented a function for deleting a task from the array.

- Implemented a function for editing task descriptions.

- All changes to the To Do List are saved in local storage.

### CODE REVIEW

File: src/modulus/addlist.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+// Add a TODO
+import task from './list.js';
+import { input, list } from './getelements.js';
+
+const add = () => {
+  // add object to list
+  const { value } = input;
+  const object = { description: value, complete: false, index: task.length };
+  if (localStorage.getItem('ls') === null) {
```
Comment (zainsadaqat) [inline src/modulus/addlist.js:12]: When I refresh the page, All added todo items get disappeared although the items are still in the local storage. Kindly fix this issues.
#### Screenshot
Screenshot 2022-08-12 at 9 48 26 PM

File: src/modulus/editD.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+// Edit task discription
+import task from './list.js';
+
+const edit = (txtchange, id) => {
+  const vl = txtchange.value;
+  task[id].description = vl;
+  localStorage.setItem('ls', JSON.stringify(task));
+};
```
Comment (zainsadaqat) [inline src/modulus/editD.js]: Edit feature isn't working. It does show that I've edited a todo item but changes are not reflected to the local storage. There's still old todo item although I already edited that on the interface. Kindly fix this issue 👍🏻 
#### Screenshot
Screenshot 2022-08-12 at 9 51 19 PM

**Review by zainsadaqat — CHANGES_REQUESTED** [review]
# Required Changes 🔁
# Hello @Suzi216  😄 

You're doing a great job on your project. We're proud of you. Good job so far! 

There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
  ✅  Correct use of Github Flow
  ✅  No linter errors
  ✅  Descriptive Pull Request

## Required Changes ♻️

_Check the comments under the review._




## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @Suzi216, 

### Good job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Clean/Organized code

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**